pw_scale_bootstrp.m

```diff
@@ -80,7 +85,13 @@
 N = sqrt( size(MM,2) );
 
 M = reshape( sum(MM,1), N, N );
-[jod, R] = pw_scale( round(M) );
+
+% Ties: Round the results to the smaller or larger integer (randomly) but 
+% agreeing with the number of comparisons.
```

Comment:
I am not getting the purpose of this piece of code - we may need to check with Maria. 
We do not handle ties